src/items/rqgItem.ts

```diff
@@ -121,22 +121,26 @@ export class RqgItem extends Item {
     const { parent, pack, ...options } = context;
     if (parent?.documentName === "Actor") {
       updates.forEach((u) => {
-        const document = parent.items.get(u._id);
-        if (!document || document.documentName !== "Item") {
-          const msg = "couldn't find item document from result";
-          ui.notifications?.error(msg);
-          throw new RqgError(msg, updates);
+        if (u && u.length > 0) {
+          const document = parent.items.get(u._id);
+          if (!document || document.documentName !== "Item") {
+            const msg = "couldn't find item document from result";
+            ui.notifications?.error(msg);
+            throw new RqgError(msg, updates);
+          }
+          // Will update "updates" as a side effect
+          ResponsibleItemClass.get(document.data.type)?.preUpdateItem(
+            parent,
+            document,
+            updates,
+            options
+          );
         }
-        // Will update "updates" as a side effect
-        ResponsibleItemClass.get(document.data.type)?.preUpdateItem(
-          parent,
-          document,
-          updates,
-          options
-        );
       });
     }
-    return super.updateDocuments(updates, context);
```

Comment:
The problem is here, but it's going to throw an exception when you delete a cult unless you do the if (updates[0].length > 0) check, BUT, for some reason, if you do that check then it doesn't add new spells to the cult.